design-documents/catalog-domain/catalog-service.md

```diff
@@ -0,0 +1,204 @@
+# Catalog Service
+
+## Purpose of Catalog Service
+Catalog has a single purpose of returning product information that was intended to be shown at the storefront, both product listing or PDP.
+Basically saying Catalog will answer on questions:
+* How the product should look like to satisfy storefront requirements.
+* How client of the Catalog can request products.
+
+## Usage examples
+
+### Direct call
+Catalog service can be involved directly through the gateway.
+![Dirrect call](images/catalog-usage-01-01.png)
+
+A good example of such services is sales channels integrations that have to synchronize across connected channels.
+
+### Routing
+Clients who want to retrieve product data by URL.
```

Comment:
Is this example of PDP?

Reply:
yes, or storefront integration that wants to get information about product storefront representation. for instance recommendation service or google merchant center